app.js

```diff
@@ -21,6 +22,13 @@ setupMiddleware(app);
 // registerRoutes(app);
 app.use(routes);
 
+// Set templating engines
+app.set('view engine', 'html'); // set default engine as "html"
+app.set('views', path.join(__dirname, './app/views')); // set path
+// app.set('views', path.join(__dirname, 'views'));
```

Comment:
Please remove the useless comments before merging for cleaner codebase.

Reply:
ok